pkg/presto/util.go

```diff
@@ -149,7 +149,7 @@ func generateCreateTableSQL(catalog, schema, tableName string, columns []Column,
 )
 %s
 %s`
-	return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, propsStr)
+	return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, propsStr, comment)
```

Comment:
comment comes before `propsStr` and we need to `prefix `COMMENT` to the message if set.

Eg:
```
commentStr := ""
if comment != "" {
  commentStr = "COMMENT " + comment
}

...
return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, commentStr, propsStr)
```

Reply:
Got it! Thank you very much.

Reply:
Got it! Thank you very much.